client/src/entrypoints/admin/page-editor.js

```diff
@@ -1,33 +1,33 @@
 import $ from 'jquery';
-import { cleanForSlug } from '../../utils/text';
+// import { cleanForSlug } from '../../utils/text';
 import { InlinePanel } from '../../components/InlinePanel';
 import { MultipleChooserPanel } from '../../components/MultipleChooserPanel';
 
 window.InlinePanel = InlinePanel;
 window.MultipleChooserPanel = MultipleChooserPanel;
-window.cleanForSlug = cleanForSlug;
+// window.cleanForSlug = cleanForSlug;
 
-function initSlugAutoPopulate() {
-  let slugFollowsTitle = false;
+// function initSlugAutoPopulate() {
+//   let slugFollowsTitle = false;
```

Comment:
let's remove this stuff completely, it's really hard to read the diff when it's commented out.